cdap-standalone/src/main/resources/logback.xml

```diff
@@ -16,18 +16,15 @@
   the License.
   -->
 
-<!--                                                                         -->
-<!-- AppFabric Logging Configuration                                         -->
-<!--                                                                         -->
-<!-- We use the LogBack project for logging in AppFabric. The manual for     -->
-<!-- Logback can be found here: http://logback.qos.ch/manual                 -->
-<!--                                                                         -->
+<!--                                                                                      -->
+<!-- AppFabric Logging Configuration                                                      -->
```

Comment:
I think this should not be AppFabric. It applies to all CDAP system services that run in master or in edge nodes.